cmd/updater/main.go

```diff
@@ -15,5 +15,7 @@ import (
 )
 
 func main() {
+	// root user is changed to dd-agent to avoid permission issues
+	rootToDDAgent()
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 Praise:** ‏Super cool feature!